### ISSUE TITLE
making installer failed

### ISSUE THREAD

**Issue by pjljvandelaar** [TorXakis/SourceWindowsInstaller#33]
See https://ci.appveyor.com/project/torxakis-admin/sourcewindowsinstaller/builds/21382947
```
Creating "v0.8.0" release for repository "TorXakis/TorXakis" tag "v0.8.0"...Error creating GitHub release: Provider setting not found or it's value is empty. If secure setting is used please check that value was encrypted (or YAML was exported) while being logged under correct account.
24380
Parameter name: auth_token
```

**Comment by torxakis-admin** [comment]
I created a token for torxakis-admin and used it in https://github.com/TorXakis/SourceWindowsInstaller/blob/develop/.appveyor.yml
However, is that correct? In the webhook I see the identity of the pusher (@pjljvandelaar): 
should the secure auth_token be his?

**Comment by keremispirli** [comment]
According to AppVeyor documentation, we need to create a token from "https://github.com/settings/tokens" and encrypt it using “Encrypt configuration data” page in AppVeyor (Settings → Encrypt YAML).
See: https://www.appveyor.com/docs/deployment/github/#provider-settings
So if this token is a Personal Access Token for torxakis-admin user and encrypted properly, then it should be alright.

But when I check the TorXakis releases, I see a TorXakis.msi for v0.8.0 so it looks like it worked despite the error. Was that with my token or the new token? Can you try to create other versions with this new token to verify?

**Comment by pjljvandelaar** [comment]
The v0.8.0 release was manually added.

The token was for right for TorXakis-admin, yet the change (adding the 0.8.0 tag) was done by a developer. Should TorXakis-admin also make the change for the new tag?

**Comment by keremispirli** [comment]
I see that the build failed at 16:23 and new token was added at 16:58. Can you try to make another release with the new token?
It should not matter who tags the release, but we can try as many times as we want since the created releases are always Draft and both the release and the tag can be deleted.

**Comment by pjljvandelaar** [comment]
O, i  though tags could not be removed...

**Comment by keremispirli** [comment]
See here: https://stackoverflow.com/a/5480292
You can just delete the tag in the remote and push it again to trigger another build.

**Comment by keremispirli** [comment]
I just successfully created a v0.8.0.1 draft release with installer. I pushed the tags with my own user.
https://ci.appveyor.com/project/torxakis-admin/sourcewindowsinstaller/builds/22789443
I'm deleting the tags and the draft release now.

**Comment by pjljvandelaar** [comment]
Great!
I don't understand why it went wrong earlier.
But it seems everything is ready for the next release....

**Comment by pjljvandelaar** [comment]
I just saw that our caching is not ok
```
Cache entry not found: C:\Users\appveyor\AppData\Local\Programs\stack\x86_64-windows\msys64
Cache entry not found: C:\Users\appveyor\AppData\Local\Programs\stack\x86_64-windows\msys64.installed
```

**Comment by keremispirli** [comment]
I saw that too. The extraction path seems to match the configured path, but still fails. I will look into it.

**Comment by pjljvandelaar** [comment]
Thanks!

**Comment by pjljvandelaar** [comment]
changed to `%LOCALAPPDATA%\Programs\stack\x86_64-windows\msys2-20180531` to fix the caching issue

**Comment by pjljvandelaar** [comment]
This issue is solved, since 0.8.1 was automatically released!
